.github/workflows/container-build.yml

```
@@ -43,38 +43,24 @@ jobs:
             git checkout origin/master -- Dockerfile
           fi
 
-      - name: Build and cache first build stage
-        # update the first stage cache only when pushing to or tagging main
-        if: github.event_name == 'push'
-        uses: docker/build-push-action@v1
-        env:
-          # use BuildKit to speed up builds and improve caching
-          DOCKER_BUILDKIT: 1
+      - name: Setup Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@v3
         with:
-          username: ${{ github.repository_owner }}
-          password: ${{ secrets.CR_PAT }}
-          registry: ${{ env.GHCR_HOSTNAME }}
-          repository: ${{ github.repository }}
-          target: build
-          tags: build
-          build_args: BUILDKIT_INLINE_CACHE=1
-          cache_froms: ${{ env.GHCR_HOSTNAME }}/${{ github.repository }}:build
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
```

```suggestion
          password: ${{ secrets.CR_PAT}}
```
I don't know if transcend-io has GITHUB_TOKEN defined, so we should change back to the original. You don't have to make this change, I can make this change once I merge in the PR.

---

Actually, looks like GITHUB_TOKEN is always defined in all github repos?

https://docs.github.com/en/actions/security-guides/automatic-token-authentication#modifying-the-permissions-for-the-github_token

---

Yes, and it should have the ability to push to GHCR, no reason to use a PAT for this.